playbooks/TIM Automation/collection.metadata.json

```diff
@@ -1,7 +1,7 @@
 {
     "@context": "\/api\/3\/contexts\/WorkflowCollection",
     "@type": "WorkflowCollection",
-    "name": "Workspace Automation",
+    "name": "TIM Automation",
```

Comment:
why is the report file below deleted?

Reply:
Its not deleted, it moved in another folder.